scripts/download-rc.js

```diff
@@ -92,13 +92,13 @@ const downloadRC = () => {
             console.log('hazelcast-enterprise.jar already exists, not downloading from maven.');
         } else {
             console.log('Downloading: hazelcast enterprise jar '
-                + `com.hazelcast:hazelcast-enterprise:${HAZELCAST_ENTERPRISE_VERSION}`);
+                + `com.hazelcast:hazelcast-enterprise-all:${HAZELCAST_ENTERPRISE_VERSION}`);
```

Comment:
Why do we need all jar?

Reply:
when enterprise jar is used `Error: Cannot execute SQL query because "hazelcast-sql" module is not in the classpath.
` is thrown